### PR TITLE
i3-open-next-ws: init at 0.1.5

### PR DESCRIPTION
**Pull request by quantenzitrone**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [x] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by quantenzitrone** [comment]
Result of `nixpkgs-review pr 273418` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>i3-open-next-ws</li>
  </ul>
</details>

**Comment by quantenzitrone** [comment]
just changes to the meta attributes

File: pkgs/by-name/i3/i3-open-next-ws/package.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  lib,
+  rustPlatform,
+  fetchCrate,
+  ...
```
Comment (SuperSandro2000) [inline pkgs/by-name/i3/i3-open-next-ws/package.nix:5]:
```suggestion
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Review by SuperSandro2000 — APPROVED** [review]
LGTM but didn't try out

**Review by pbsds — APPROVED** [review]
Result of `nixpkgs-review pr 273418` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>i3-open-next-ws</li>
  </ul>
</details>

`--help` seems to work, but it panics if sway is not found. I have not tested with sway.

<details>

```
[nix-shell:/dev/shm/nixpkgs-review/pr-273418]$ RUST_BACKTRACE=full results/i3-open-next-ws/bin/i3-open-next-ws --help
i3-open-next-ws 0.1.5
A companion utility to i3wm for managing workspaces. Call without arguments to get first unused
workspace number on STDOUT

USAGE:
    i3-open-next-ws [OPTIONS] [WORKSPACE]

ARGS:
    <WORKSPACE>    overrides workspace selection

OPTIONS:
    -e, --exec <COMMAND>    Execute COMMAND on a new workspace
    -f, --focus             Focus first unused workspace
    -h, --help              Print help information
    -m, --move              Move focused window to the first unused workspace
    -V, --version           Print version information

[nix-shell:/dev/shm/nixpkgs-review/pr-273418]$ RUST_BACKTRACE=full results/i3-open-next-ws/bin/i3-open-next-ws
thread 'main' panicked at /build/i3-open-next-ws-0.1.5-vendor.tar.gz/swayipc/src/socket.rs:17:9:
internal error: entered unreachable code
stack backtrace:
   0:     0x55fec498bb0b - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::h580634b3cf1e80d2
   1:     0x55fec49a751c - core::fmt::write::h1907008f375a0f7f
   2:     0x55fec496e17e - std::io::Write::write_fmt::h5d182c40a724ef6c
   3:     0x55fec498b8e5 - std::sys_common::backtrace::print::hb1b7da1f58c2e37d
   4:     0x55fec498cdc3 - std::panicking::default_hook::{{closure}}::h7f74968e4c453ed8
   5:     0x55fec498cac5 - std::panicking::default_hook::h1b091f0aa6d8371d
   6:     0x55fec498d355 - std::panicking::rust_panic_with_hook::h951c7ebb95d3bb72
   7:     0x55fec498d1f6 - std::panicking::begin_panic_handler::{{closure}}::hc652a55312dedae5
   8:     0x55fec498bd16 - std::sys_common::backtrace::__rust_end_short_backtrace::h7eaedff2f8f958bb
   9:     0x55fec498cf92 - rust_begin_unwind
  10:     0x55fec48bbd05 - core::panicking::panic_fmt::h2e7bf6f6dde740cb
  11:     0x55fec48bbda3 - core::panicking::panic::hd4dfdca7f0c595c1
  12:     0x55fec48cf122 - swayipc::socket::get_socketpath::ha1f482b9a06b0c37
  13:     0x55fec48ce187 - swayipc::connection::Connection::new::h19066cf57c143341
  14:     0x55fec48c4c19 - i3_open_next_ws::main::h6b9c82c2dd5d63d3
  15:     0x55fec48cb7cd - std::sys_common::backtrace::__rust_begin_short_backtrace::h75a064b999dfdf3f
  16:     0x55fec48cb205 - std::rt::lang_start::{{closure}}::h4e0bfa206e62313a
  17:     0x55fec496cd47 - std::rt::lang_start_internal::he83d9963a9dcb9b1
  18:     0x55fec48c6d65 - main
  19:     0x7fea8e617fce - __libc_start_call_main
  20:     0x7fea8e618089 - __libc_start_main_impl
  21:     0x55fec48bc555 - _start
  22:                0x0 - <unknown>
```
</details>



- [x] package path fits guidelines
- [x] package name fits guidelines
- [x] package version fits guidelines
- [x] package builds on x86_64-linux
- [x] executables tested on x86_64-linux
- [x] `meta.description` is set and fits guidelines
- [x] `meta.license` fits upstream license
- [x] `meta.platforms` is set
- [x] `meta.maintainers` is set
- [x] build time only dependencies are declared in `nativeBuildInputs`
- [x] source is fetched using the appropriate function
- [x] the list of `phases` is not overridden
- [x] when a phase (like `installPhase`) is overridden it starts with `runHook preInstall` and ends with `runHook postInstall`.
- [x] patches have a comment describing either the upstream URL or a reason why the patch wasn't upstreamed
- [x] patches that are remotely available are fetched rather than vendored


LGTM

**Pull request merged**